app/Providers/AppServiceProvider.php

```diff
@@ -29,6 +29,11 @@ public function boot()
 
         View::share('appVersion', $this->versionData()['version'] ?? 'undefined');
         View::share('appIsGit', $this->versionData()['is_git'] ?? false);
+        $googleAnalyticsId = env('APP_GOOGLE_ANALYTICS_ID');
+        if ($googleAnalyticsId) {
+            View::share('appGoogleAnalyticsId', $googleAnalyticsId);
```

Comment:
I'd be fine with `$analyticsId` as the variable name here, and you can just always pass it through and make the blade do a `!empty($analyticsId)` check.